data/resources.yml

```diff
@@ -271,3 +271,114 @@
+  tags:
+  - Tools
+  title: NASA WorldView (EOSDIS)
+  url: https://worldview.earthdata.nasa.gov/
+- description: Search job openings across the Breakthrough Energy Ventures network, an initiative founded by Bill Gates.
+  tags:
+  - Climate Careers
+  title: Breakthrough Energy Ventures job board
+  url: https://bevjobs.breakthroughenergy.org/jobs
+- description: Search job openings across the Amazon Climate Pledge network.
+  tags:
+  - Climate Careers
+  title: Climate Pledge Fund jobs site
+  url: https://portfoliojobs.theclimatepledge.com/jobs
+- description: Veteran climate journalist Yessenia Funes is ready to remind you that the warming of the world is unjust, and that protecting the planet must include its people, in the flagship features newsletter of the environmental justice magazine Atmos.
+  tags:
+  - Climate Journalism
+  - Education
+  - Climate Accountability
+  title: The Frontline by Atmos magazine
+  url: https://atmos.earth/topic/environmental-justice-newsletter-frontline/
+- description: Ecosia uses the ad revenue from your searches to plant trees where they're needed most. Also has tooling for quickly spotting websites belonging to climate friendly, and climate polluting companies.
+  tags:
+  - Tools
+  title: Ecosia
+  url: https://ecosia.org/
+- description: This site makes it easier for Canadians to keep track of our federal representatives, allowing people to easily visualize, search and follow what has been and is being said and done by Members of Parliament.
+  tags:
+  - Politics/Canada
+  title: openparliament.ca
+  url: https://openparliament.ca/
+- description: Compilation of interesting climate-related PDFs by Rollie and the Climate Town staff. Updated every season.
+  tags:
+  - 🍞 Rollie's Choice
+  - Climate Accountability
+  - Education
+  - Research
+  title: A Top List of Interesting PDFs
+  url: https://www.climatetownproductions.com/favoritepdfs
+- description: A report and database of open source technology projects to sustain a stable climate, energy supply, biodiversity and natural resources. Lists and analyses over 1300 different projects.
+  tags:
+  - ✨ Mod Choice
+  - 📚 Knowledge Hubs
+  - Climate Tech
+  title: Open Source in Environmental Sustainability
+  url: https://report.opensustain.tech/chapters/index.html
+- description: A database of information and case documents for over 1000 climate change lawsuits from around the world, hosted by the Sabin Center for Climate Change Law at Columbia University.
+  tags:
+  - Climate Accountability
+  - Research
+  title: Climate Case Chart, by the Sabin Center
+  url: http://climatecasechart.com/
+- description: The Climate Attribution Database is a virtual library hosted by Columbia University’s Earth Institute and Sabin Center for Climate Change Law, and centered on climate change aimed at law experts. It contains over 500 scientific resources, organized under four thematic umbrellas.
```

Comment:
Add period after "Change Law". Replace rest of description with "It contains over 500 scientific resources organized under four thematic umbrellas centered on climate change, and is aimed at law experts."